web_widget_image_url/static/src/xml/web_widget_image_url.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+-->
+<templates xml:space="preserve">
+    <t t-name="FieldImageURL">
+        <span class="oe_form_field oe_form_field_image" t-att-style="widget.attrs.style">
+            <img t-att-src="widget.url()"
+            t-att-border="widget.readonly ? 0 : 1"
```

Comment:
The border attribute is not deprecated in html5.

Reply:
I don't get it. Could you please describe it more?

Reply:
The attribute border is deprecated, please remove it. You can use the style attribute (t-att-style) to add a border to the image. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Deprecated_attributes

Reply:
Thanks for the explanation.